composer.json

```diff
@@ -4,6 +4,9 @@
         "ext-gd": "0",
         "ext-curl": "0"
     },
+    "suggest": {
+        "ext-imagick": "to use the Imagick implementation",
```

Comment:
in fact, you’re using `Imagick` classes, it should be a requirement. Or use dependency inversion.